ui/DesignSystem/Component/ProjectList.svelte

```diff
@@ -1,19 +1,22 @@
-<script>
-  import { BadgeType } from "../../src/badge.ts";
+<script lang="ts">
+  import { BadgeType } from "../../src/badge";
+  import * as project from "../../src/project";
```

Comment:
It's not straightfoward to use `interface`s as named imports with rollup right now - see https://github.com/rollup/plugins/issues/481

I personally don't mind sticking with this ESM syntax for now:
```typescript
import * as project from "../../src/project";
```

but I wanted to mention this in case someone gets an error like:
```
[!] Error: 'Project' is not exported by ui/src/project.ts, imported by ui/DesignSystem/Component/ProjectList.svelte
```

when importing this way:
```typescript
import { Project } from "../../src/project";
```